### DISCUSSION TITLE
Not getting data from a visible peer

### DISCUSSION THREAD

**Discussion by despens** [beakerbrowser/beaker#1837] [Q&A]
I was sent a hyper:// URL by a peer currently online. Beaker reports the hyperdrive as "not found", but shows "1 peer" being connected, and the correct IP address of the peer.

What could be the issue here?
Does the remote network need to be hole-punchable? (And does that mean UPnP being activated?) 
Is opening port 3282 sufficient?

**Comment by yhancik** [comment]
The "hole punchable" status really deserves to be more clear to users. I doesn't really say where the problem might lie (is it a computer config, a router feature?) or what can be done (or not) about it.

**Comment by despens** [comment]
Yes indeed.

BTW, the connection worked once the remote peer was moved behind a router that allowed UPnP. 

Still it seems weird that the peer is visible in a non-UPnP network, but no data transfer can happen.
